### ISSUE TITLE
bind options from one mesh affect the other mesh config

### ISSUE THREAD

**Issue by igarny** [perfsonar/mesh-config#96]
Hi Andy,
On my RRZE host I have activated 2 meshes. One is the pS Dev mesh and the other mesh for 2 private network hosts.
The problem I observe is that with current nightly release and my meshconfig automatically added a global bind option to the -tasks.conf:
```
<bind_address>
    bind_address   10.10.1.2
    added_by_mesh   1
</bind_address>
```

My expectations are, that this will impact immediately the pSDev Mesh.
In the pS 4.0 version, this behavior was not present: see #64 and the json file (url)

My tests and effort are in-line with the #64

**Comment by igarny** [comment]
local json file is: https://t2-psdev.rrze.uni-erlangen.de/rrzemesh.json

**Comment by igarny** [comment]
it appears my concerns are largely have no base. I have found most of the measurements working after a full day of mesh operation.
There are glitches with specific test specifications, which deserve a special attention/ticket.
I have found a working:
local-2-remote tests
- latencybg
- iperf3
- rtt
- traceroute

remote-2-local tests
- latencybg
- iperf3
- traceroute
- rtt

I have found problematic: rtt tests, being skipped by meshconfig. see #97
